extensions/functions/src/main/java/io/crate/operation/aggregation/HyperLogLogDistinctAggregation.java

```diff
@@ -352,6 +451,8 @@ long hash(Object val) {
 
         static class Bytes extends Murmur3Hash {
 
+            static final Bytes INSTANCE = new Bytes();
```

Comment:
Please revert - this won't be thread-safe because the `MurmurHash3.Hash128` is mutated. 

My earlier comment was probably not clear. I meant that `MurmurHash3.Hash128 hash = new MurmurHash3.Hash128();` can be done within the specialized `DocValueAggregator` instance - not that we can re-use a `Bytes` instance.